tests/unit/splinter_test.c

```diff
@@ -668,6 +722,7 @@ splinter_do_inserts(void         *datap,
    // Debug hook: Override this to smaller value for faster test execution,
    // while doing test-dev / debugging. Default is some big value, like
    // 12988800; num_inserts = (100 * 100 * 100);
+   // num_inserts = (100 * 100 * 100);
```

Comment:
Rebase artifact?

Reply:
Not exactly.

In another review, you had frowned upon the use of variables named `shadowp` or `shadowpp`.

The code had been fixed, previously, but the prolog comment was not. I am just fixing the name `shadowpp`.

(Btw ..., as an aside, it does help with understanding to see `shadowpp`, especially if it's an output parameter, where caller is passing-in addr-of-ptr-to-struct to be filled-out.)

Reply:
I missed the line you remarked on. Deleted L725. I had added it for my own dev-testing.

Reply:
I was referring to the some expression on L723.